boa_parser/src/lexer/cursor.rs

```diff
@@ -13,33 +13,31 @@ pub(super) struct Cursor<R> {
 
 impl<R> Cursor<R> {
     /// Gets the current position of the cursor in the source code.
-    #[inline]
+
```

Comment:
Don't like the empty line between the comment and function
There's a few more in the file